### PR TITLE
zh-CN: site/zh-cn/beta/tutorials/load_data/text.ipynb

### PR DESCRIPTION
**Pull request by KGBMAX**

### CODE REVIEW

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/r2/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]: 55-70行， 请参考 https://docs.google.com/document/d/1v7kwEhvtbbI4S4gulxLJDDN78gv0ZlDJ0cN92BUPrm0 修改，注意将按钮和链接汉化并修改。

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+   "source": [
+    "## 将文本编码成数字\n",
+    "\n",
+    "机器学习基于的是数字而非文本，所以字符串需要被转化成数字列表。\n",
+    "为了达到此目的，我们需要构建文本与整数的一一映射。\n",
+    "\n",
+    "### 建立词汇表\n",
+    "\n",
+    "\n",
+    "首先，通过将文本标记为单独的单词集合来构建词汇表。在TensorFlow 和 Python 中均有很多方法来达成这一目的。在本教程中:\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "首先，通过将文本标记为单独的单词集合来构建词汇表。在 TensorFlow 和 Python 中均有很多方法来达成这一目的。在本教程中:\n",
```

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "v6S5Qyabi-vo"
+   },
+   "source": [
+    "你可以尝试运行这一行去看一下输出的样式。"
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "你可以尝试运行这一行代码并查看输出的样式。"
```

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 4]
```diff
@@ -0,0 +1,665 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "_8OJOPohKh1q"
+   },
+   "source": [
+    "\n",
+    "下一层是 [LSTM](http://colah.github.io/posts/2015-08-Understanding-LSTMs/) 层，它允许模型利用上下文中理解单词含义。 LSTM上的双向包装器有助于模型理解当前数据点与其之前和之后的数据点的关系。\n"
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "下一层是 [LSTM](http://colah.github.io/posts/2015-08-Understanding-LSTMs/) 层，它允许模型利用上下文中理解单词含义。 LSTM 上的双向包装器有助于模型理解当前数据点与其之前和之后的数据点的关系。\n"
```

**Comment by KGBMAX** [comment]
@Mr-Linus

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 5]
```diff
@@ -0,0 +1,665 @@
...
+    "colab_type": "text",
+    "id": "1ap_W4aQcgNT"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/tutorials/load_data/text\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />在 TensorFlow.org 上查看</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />在 Google Colab 上运行</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />在 Google Colab 上运行</a>\n",
```

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 6]
```diff
@@ -0,0 +1,665 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />查看 GitHub 上的资源</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />查看 GitHub 上的资源</a>\n",
```

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 7]
```diff
@@ -0,0 +1,665 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/r2/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />下载 notebook</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]:
```suggestion
    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/beta/tutorials/load_data/text.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />下载 notebook</a>\n",
```

File: site/zh-cn/beta/tutorials/load_data/text.ipynb (hunk 1 above)
Comment (Mr-Linus) [reply site/zh-cn/beta/tutorials/load_data/text.ipynb]: 免责声明还没有加

File: site/zh-cn/beta/tutorials/load_data/text.ipynb [hunk 8]
```diff
@@ -0,0 +1,678 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1ap_W4aQcgNT"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/tutorials/load_data/text\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />在 TensorFlow.org 上查看</a>\n",
```
Comment (lamberta) [inline site/zh-cn/beta/tutorials/load_data/text.ipynb]: For files in `zh-cn/`, replace all occurrences of `www.tensorflow.org` with `tensorflow.google.cn`
(here and throughout)
Thanks

Comment (KGBMAX) [reply site/zh-cn/beta/tutorials/load_data/text.ipynb]: Thanks！ Already updated

**Comment by Mr-Linus** [comment]
I didn't find this problem. My fault. Sorry 😅

**Comment by KGBMAX** [comment]
Thanks！

**Pull request merged**